Build/AdditionalProjectTransforms.xslt

```diff
@@ -0,0 +1,5 @@
+<xsl:if test="/Input/Properties/DefineNET4DetectionConstant = 'True'">
+  <PropertyGroup>
+    <DefineConstants Condition=" '$(TargetFrameworkVersion)' == 'v4.0' ">NET4</DefineConstants>
```

Comment:
Not that I care too much, but surely the number of platforms that are built on .NET 4.0 is pretty limited now... is there a reason to define `NET45` with this conditional magic instead of just appending it to the platforms that are .NET 4.5 and higher?